pkg/envoy/config/v2/bootstrap.go

```diff
@@ -67,7 +67,7 @@ func GenerateBootstrapConfig(host string, port uint32) (string, error) {
 					{
 						TargetSpecifier: &envoy_api_v2_core.GrpcService_EnvoyGrpc_{
 							EnvoyGrpc: &envoy_api_v2_core.GrpcService_EnvoyGrpc{
-								ClusterName: "ads_cluster",
+								ClusterName: "xds_cluster",
```

Comment:
This is for consistency because we are calling the discovery service xDS everywhere.